### DISCUSSION TITLE
pnpm build not working

### DISCUSSION THREAD

**Discussion by thecynic101** [elizaOS/eliza#1264] [Q&A]
Noob here. This is the error I keep getting after running pnpm build. I dunno what to do. : 

"> eliza@ build /home/the_cyn
> eliza@ build /home/the_cynic/work/ai-agent/eliza
> turbo run build --filter=!
> turbo run build --filter=!eliza-docs

turbo 2.3.3

• Packages in scope: @ai16z/
• Packages in scope: @ai16z/adapter-postgres, @ai16z/ada
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supaba
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/cli
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-dire
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai1
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/cli
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twit
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/pl
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plu
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plu
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow,
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow, @ai16z/plugin-goat, @ai16z/
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow, @ai16z/plugin-goat, @ai16z/plugin-icp, @ai16z/plugin-im
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow, @ai16z/plugin-goat, @ai16z/plugin-icp, @ai16z/plugin-image-generation, @ai16z/plugi
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow, @ai16z/plugin-goat, @ai16z/plugin-icp, @ai16z/plugin-image-generation, @ai16z/plugin-intiface, @ai16z/plugin-no
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow, @ai16z/plugin-goat, @ai16z/plugin-icp, @ai16z/plugin-image-generation, @ai16z/plugin-intiface, @ai16z/plugin-node, @ai16z/plugin-solana, @a
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow, @ai16z/plugin-goat, @ai16z/plugin-icp, @ai16z/plugin-image-generation, @ai16z/plugin-intiface, @ai16z/plugin-node, @ai16z/plugin-solana, @ai16z/plugin-starknet, @ai16z
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow, @ai16z/plugin-goat, @ai16z/plugin-icp, @ai16z/plugin-image-generation, @ai16z/plugin-intiface, @ai16z/plugin-node, @ai16z/plugin-solana, @ai16z/plugin-starknet, @ai16z/plugin-tee, @ai16z/plugin-t
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow, @ai16z/plugin-goat, @ai16z/plugin-icp, @ai16z/plugin-image-generation, @ai16z/plugin-intiface, @ai16z/plugin-node, @ai16z/plugin-solana, @ai16z/plugin-starknet, @ai16z/plugin-tee, @ai16z/plugin-trustdb, @ai16z/plugin-video-
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow, @ai16z/plugin-goat, @ai16z/plugin-icp, @ai16z/plugin-image-generation, @ai16z/plugin-intiface, @ai16z/plugin-node, @ai16z/plugin-solana, @ai16z/plugin-starknet, @ai16z/plugin-tee, @ai16z/plugin-trustdb, @ai16z/plugin-video-generation, @ai16z/plugin-we
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow, @ai16z/plugin-goat, @ai16z/plugin-icp, @ai16z/plugin-image-generation, @ai16z/plugin-intiface, @ai16z/plugin-node, @ai16z/plugin-solana, @ai16z/plugin-starknet, @ai16z/plugin-tee, @ai16z/plugin-trustdb, @ai16z/plugin-video-generation, @ai16z/plugin-web-search, @ai16z/plugin-what
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow, @ai16z/plugin-goat, @ai16z/plugin-icp, @ai16z/plugin-image-generation, @ai16z/plugin-intiface, @ai16z/plugin-node, @ai16z/plugin-solana, @ai16z/plugin-starknet, @ai16z/plugin-tee, @ai16z/plugin-trustdb, @ai16z/plugin-video-generation, @ai16z/plugin-web-search, @ai16z/plugin-whatsapp, create-eliza-app, eliz
• Packages in scope: @ai16z/adapter-postgres, @ai16z/adapter-sqlite, @ai16z/adapter-sqljs, @ai16z/adapter-supabase, @ai16z/agent, @ai16z/client-auto, @ai16z/client-direct, @ai16z/client-discord, @ai16z/client-farcaster, @ai16z/client-github, @ai16z/client-linkedin, @ai16z/client-telegram, @ai16z/client-twitter, @ai16z/eliza, @ai16z/plugin-0g, @ai16z/plugin-aptos, @ai16z/plugin-bootstrap, @ai16z/plugin-coinbase, @ai16z/plugin-conflux, @ai16z/plugin-echochambers, @ai16z/plugin-evm, @ai16z/plugin-flow, @ai16z/plugin-goat, @ai16z/plugin-icp, @ai16z/plugin-image-generation, @ai16z/plugin-intiface, @ai16z/plugin-node, @ai16z/plugin-solana, @ai16z/plugin-starknet, @ai16z/plugin-tee, @ai16z/plugin-trustdb, @ai16z/plugin-video-generation, @ai16z/plugin-web-search, @ai16z/plugin-whatsapp, create-eliza-app, eliza-client
• Running build in 36 packag
• Running build in 36 packages
• Remote caching disabled
create-eliza-app:build: cach
create-eliza-app:build: cache miss, executing b28782f438
create-eliza-app:build: cache miss, executing b28782f438f0daaf
@ai16z/eliza:build: cache mi
@ai16z/eliza:build: cache miss, executing 71193ed6fe4dcd
@ai16z/eliza:build: cache miss, executing 71193ed6fe4dcd62
create-eliza-app:build: 
@ai16z/eliza:build: 
create-eliza-app:build: 
create-eliza-app:build: > cr
create-eliza-app:build: > create-eliza-app@0.1.5-alpha.5
create-eliza-app:build: > create-eliza-app@0.1.5-alpha.5 build /home/the_cynic/work/
create-eliza-app:build: > create-eliza-app@0.1.5-alpha.5 build /home/the_cynic/work/ai-agent/eliza/packages/crea
create-eliza-app:build: > create-eliza-app@0.1.5-alpha.5 build /home/the_cynic/work/ai-agent/eliza/packages/create-eliza-app
create-eliza-app:build: > un
create-eliza-app:build: > unbuild
create-eliza-app:build: 
@ai16z/eliza:build: 
@ai16z/eliza:build: > @ai16z
@ai16z/eliza:build: > @ai16z/eliza@0.1.5-alpha.5 build /
@ai16z/eliza:build: > @ai16z/eliza@0.1.5-alpha.5 build /home/the_cynic/work/ai-agent
@ai16z/eliza:build: > @ai16z/eliza@0.1.5-alpha.5 build /home/the_cynic/work/ai-agent/eliza/packages/core
@ai16z/eliza:build: > tsup -
@ai16z/eliza:build: > tsup --format esm --dts
@ai16z/eliza:build: 
@ai16z/eliza:build: ✘ [ERROR
@ai16z/eliza:build: ✘ [ERROR] Expected identifier but fo
@ai16z/eliza:build: ✘ [ERROR] Expected identifier but found "import"
@ai16z/eliza:build: 
@ai16z/eliza:build:     (def
@ai16z/eliza:build:     (define name):1:0:
@ai16z/eliza:build:       1 
@ai16z/eliza:build:       1 │ import.meta.url
@ai16z/eliza:build:         
@ai16z/eliza:build:         ╵ ~~~~~~
@ai16z/eliza:build: 
@ai16z/eliza:build: Error: B
@ai16z/eliza:build: Error: Build failed with 1 error:
@ai16z/eliza:build: (define 
@ai16z/eliza:build: (define name):1:0: ERROR: Expected i
@ai16z/eliza:build: (define name):1:0: ERROR: Expected identifier but found "import"
@ai16z/eliza:build:     at f
@ai16z/eliza:build:     at failureErrorWithLog (/home/th
@ai16z/eliza:build:     at failureErrorWithLog (/home/the_cynic/work/ai-agent/eliza/
@ai16z/eliza:build:     at failureErrorWithLog (/home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/mai
@ai16z/eliza:build:     at failureErrorWithLog (/home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/main.js:1476:15)
@ai16z/eliza:build:     at /
@ai16z/eliza:build:     at /home/the_cynic/work/ai-agent
@ai16z/eliza:build:     at /home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/
@ai16z/eliza:build:     at /home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/main.js:945:25
@ai16z/eliza:build:     at r
@ai16z/eliza:build:     at runOnEndCallbacks (/home/the_
@ai16z/eliza:build:     at runOnEndCallbacks (/home/the_cynic/work/ai-agent/eliza/no
@ai16z/eliza:build:     at runOnEndCallbacks (/home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/main.
@ai16z/eliza:build:     at runOnEndCallbacks (/home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/main.js:1316:45)
@ai16z/eliza:build:     at b
@ai16z/eliza:build:     at buildResponseToResult (/home/
@ai16z/eliza:build:     at buildResponseToResult (/home/the_cynic/work/ai-agent/eliz
@ai16z/eliza:build:     at buildResponseToResult (/home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/m
@ai16z/eliza:build:     at buildResponseToResult (/home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/main.js:943:7)
@ai16z/eliza:build:     at /
@ai16z/eliza:build:     at /home/the_cynic/work/ai-agent
@ai16z/eliza:build:     at /home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/
@ai16z/eliza:build:     at /home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/main.js:970:16
@ai16z/eliza:build:     at r
@ai16z/eliza:build:     at responseCallbacks.<computed> 
@ai16z/eliza:build:     at responseCallbacks.<computed> (/home/the_cynic/work/ai-age
@ai16z/eliza:build:     at responseCallbacks.<computed> (/home/the_cynic/work/ai-agent/eliza/node_modules/esbuil
@ai16z/eliza:build:     at responseCallbacks.<computed> (/home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/main.js:622:9)
@ai16z/eliza:build:     at h
@ai16z/eliza:build:     at handleIncomingPacket (/home/t
@ai16z/eliza:build:     at handleIncomingPacket (/home/the_cynic/work/ai-agent/eliza
@ai16z/eliza:build:     at handleIncomingPacket (/home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/ma
@ai16z/eliza:build:     at handleIncomingPacket (/home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/main.js:677:12)
@ai16z/eliza:build:     at S
@ai16z/eliza:build:     at Socket.readFromStdout (/home/
@ai16z/eliza:build:     at Socket.readFromStdout (/home/the_cynic/work/ai-agent/eliz
@ai16z/eliza:build:     at Socket.readFromStdout (/home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/m
@ai16z/eliza:build:     at Socket.readFromStdout (/home/the_cynic/work/ai-agent/eliza/node_modules/esbuild/lib/main.js:600:7)
@ai16z/eliza:build:     at S
@ai16z/eliza:build:     at Socket.emit (node:events:513:
@ai16z/eliza:build:     at Socket.emit (node:events:513:28)
@ai16z/eliza:build:     at a
@ai16z/eliza:build:     at addChunk (node:internal/strea
@ai16z/eliza:build:     at addChunk (node:internal/streams/readable:559:12)
@ai16z/eliza:build:  ELIFECY
@ai16z/eliza:build:  ELIFECYCLE  Command failed with exi
@ai16z/eliza:build:  ELIFECYCLE  Command failed with exit code 1.
@ai16z/eliza:build: ERROR: c
@ai16z/eliza:build: ERROR: command finished with error: 
@ai16z/eliza:build: ERROR: command finished with error: command (/home/the_cynic/wor
@ai16z/eliza:build: ERROR: command finished with error: command (/home/the_cynic/work/ai-agent/eliza/packages/co
@ai16z/eliza:build: ERROR: command finished with error: command (/home/the_cynic/work/ai-agent/eliza/packages/core) /home/the_cynic/work/ai-
@ai16z/eliza:build: ERROR: command finished with error: command (/home/the_cynic/work/ai-agent/eliza/packages/core) /home/the_cynic/work/ai-agent/eliza/node_modules/.bi
@ai16z/eliza:build: ERROR: command finished with error: command (/home/the_cynic/work/ai-agent/eliza/packages/core) /home/the_cynic/work/ai-agent/eliza/node_modules/.bin/pnpm run build exited (1)
@ai16z/eliza#build: command 
@ai16z/eliza#build: command (/home/the_cynic/work/ai-age
@ai16z/eliza#build: command (/home/the_cynic/work/ai-agent/eliza/packages/core) /hom
@ai16z/eliza#build: command (/home/the_cynic/work/ai-agent/eliza/packages/core) /home/the_cynic/work/ai-agent/el
@ai16z/eliza#build: command (/home/the_cynic/work/ai-agent/eliza/packages/core) /home/the_cynic/work/ai-agent/eliza/node_modules/.bin/pnpm r
@ai16z/eliza#build: command (/home/the_cynic/work/ai-agent/eliza/packages/core) /home/the_cynic/work/ai-agent/eliza/node_modules/.bin/pnpm run build exited (1)

 Tasks:    0 successful, 2 t
 Tasks:    0 successful, 2 total
Cached:    0 cached, 2 total
  Time:    3.471s 
Failed:    @ai16z/eliza#buil
Failed:    @ai16z/eliza#build

    ...Finishing writing to 
>   ...Finishing writing to 
 ERROR  run failed: command 
 ERROR  run failed: command  exited (1)
 ELIFECYCLE  Command failed 
 ELIFECYCLE  Command failed with exit code 1."

**Comment by errorsurface** [comment]
Not entirely sure but I think it's the same issue I was encountering with typescript ,  modifying the build command in `package.json` and pointing it to the index file fixed it for me:

```
"scripts": {
  "build": "tsup src/index.ts --format esm --dts"
}
```

then run `pnpm build` again
